src/python/WMCore/JobStateMachine/ChangeState.py

```diff
@@ -101,6 +125,7 @@ def __init__(self, config, couchDbName=None):
         self.getWorkflowSpecDAO = self.daofactory("Workflow.GetSpecAndNameFromTask")
 
         self.maxUploadedInputFiles = getattr(self.config.JobStateMachine, 'maxFWJRInputFiles', 1000)
+        self.fwjrLimitSize = getattr(self.config.JobStateMachine, 'fwjrLimitSize', 8 * 1000**2)
```

Comment:
We do not know how CouchDB code check document size, therefore to avoid edge case I suggest to change default value to be `8*1000**2 -1 `.

Reply:
I underrstand it to be the same mechanism as we adopted here. From their documentation:
 "serialized Erlang representation of the JSON document body".

Anyhow, we will have this answer soon, from a T0 replay.